test/src/browser-installer/chrome/index.ts

```diff
@@ -0,0 +1,86 @@
+import proxyquire from "proxyquire";
+import sinon, { type SinonStub } from "sinon";
+import type { ChildProcess } from "child_process";
+import type { runChromeDriver as RunChromeDriverType } from "../../../../src/browser-installer/chrome";
+
+describe("browser-installer/chrome", () => {
+    const sandbox = sinon.createSandbox();
+
+    let runChromeDriver: typeof RunChromeDriverType;
+
+    let pipeLogsWithPrefixStub: SinonStub;
+    let installChromeStub: SinonStub;
+    let installChromeDriverStub: SinonStub;
+    let spawnStub: SinonStub;
+
+    let getPortStub: SinonStub;
+    let waitPortStub: SinonStub;
+
+    beforeEach(() => {
+        pipeLogsWithPrefixStub = sandbox.stub();
+        installChromeStub = sandbox.stub().resolves("/browser/path");
+        installChromeDriverStub = sandbox.stub().resolves("/driver/path");
+        spawnStub = sandbox.stub().returns({ kill: sandbox.stub() });
+        getPortStub = sandbox.stub().resolves(12345);
+        waitPortStub = sandbox.stub().resolves();
+
+        runChromeDriver = proxyquire("../../../../src/browser-installer/chrome", {
```

Comment:
it might be better to test this functionality with integration tests. There are too many stubs right now

Reply:
Yes, everything is stubbed, because it is not important at `chrome/index.ts` level
We only need to know `installChromeDriver` is called/not called here, as well as `spawn` is called/not called.

`installChromeDriver` is tested in another test file, and none of those stubs contains any important logic

I still think those tests do test `chrome/index.ts` functionality (and nothing more)